### ISSUE TITLE
Newest PyPI release not working on Google Colab

### ISSUE THREAD

**Issue by malloyca** [MTG/essentia#1328]
Hello,

I just wanted to let you know that the newest release, 2.1b1.de1032, isn't importing correctly on Google Colab.

I'm installing and importing it with:
```
!pip install essentia
import essential.standard as es
```
The output for that is:
```
Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Collecting essentia
  Downloading essentia-2.1b6.dev1032-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (13.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 13.7/13.7 MB 78.2 MB/s eta 0:00:00
Requirement already satisfied: pyyaml in /usr/local/lib/python3.10/dist-packages (from essentia) (6.0)
Requirement already satisfied: numpy>=1.8.2 in /usr/local/lib/python3.10/dist-packages (from essentia) (1.22.4)
Requirement already satisfied: six in /usr/local/lib/python3.10/dist-packages (from essentia) (1.16.0)
Installing collected packages: essentia
Successfully installed essentia-2.1b6.dev1032
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
[<ipython-input-5-a9ecca07ae37>](https://6q1l7rjfwth-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230426-060109-RC01_527296767#) in <cell line: 2>()
      1 get_ipython().system('pip install essentia')
----> 2 import essentia.standard as es

[/usr/local/lib/python3.10/dist-packages/essentia/__init__.py](https://6q1l7rjfwth-496ff2e9c6d22116-0-colab.googleusercontent.com/outputframe.html?vrz=colab-20230426-060109-RC01_527296767#) in <module>
----> 1 from . import _essentia
      2 import sys as _sys
      3 from ._essentia import reset
      4 from .common import Pool, array, ones, zeros
      5 from .progress import Progress

RuntimeError: module compiled against API version 0x10 but this version of numpy is 0xf . Check the section C-API incompatibility at the Troubleshooting ImportError section at https://numpy.org/devdocs/user/troubleshooting-importerror.html#c-api-incompatibility for indications on how to solve this problem .
```

I found a workaround by installing an older release. I'm fine using this version, but thought you should know about this issue.
```!pip install essentia==2.1b6.dev858```
Pip didn't want to install release 2.1b6.dev871.

Thank you.

**Comment by zuble** [comment]
found the same issue with essentia-tensorflow

```
Python 3.8.15 | packaged by conda-forge | (default, Nov 22 2022, 08:46:39) 
[GCC 10.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import essentia
Unable to import NumPy C API from Essentia module. Error code = -1
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jtstudents/anaconda3/envs/zugpu2/lib/python3.8/site-packages/essentia/__init__.py", line 1, in <module>
    from . import _essentia
RuntimeError: module compiled against API version 0x10 but this version of numpy is 0xd . Check the section C-API incompatibility at the Troubleshooting ImportError section at https://numpy.org/devdocs/user/troubleshooting-importerror.html#c-api-incompatibility for indications on how to solve this problem .

numpy                     1.19.5                   pypi_0    pypi
essentia-tensorflow       2.1b6.dev1032            pypi_0    pypi
```

**Comment by dbogdanov** [comment]
Should be fixed in 7cd1a7d87dd13ef4ad692e489d0ae220107a49bb and we've pushed new wheels to PyPI. Let me know if you have any problems.

**Comment by malloyca** [comment]
Working for me on Colab now. Thanks!
